packages/editor/src/components/editor/palette/Palette.css

```diff
@@ -1,5 +1,9 @@
-.palette-category-items {
-  display: flex;
-  flex-wrap: wrap;
-  gap: var(--size-1);
+.palette {
+  width: 100%;
+}
+
+.palette-items-grid {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
```

Comment:
fine for me if you want to use grid here, but technically I think you can also use flex with wrap activated :)

Reply:
I change it back to flex. Initially I had some problem with the width of the Popover but I managed to figure it out👍